Dynatrace/dynatrace-operator
# Add LogModule configuration options for the Operator

## Description

[K8S-10533](https://dt-rnd.atlassian.net/browse/K8S-10533)

(Re)introduce the `LogModule` spec in the DynaKube.

Only adds the type and update/generate the necessary files (api-ref, deepcopy, crd)

## How can this be tested?
Check the naming/descriptions.
Run the api-ref/deepcopy/crd generation locally, if you really want to 😅
